docs/tables/aws_elb_access_log/queries.md

```diff
@@ -0,0 +1,268 @@
+## Activity Examples
+
+### Daily request trends
+
+Count requests per day to identify traffic patterns over time.
+
+```sql
+select
+  strftime(timestamp, '%Y-%m-%d') as request_date,
+  count(*) as request_count
+from
+  aws_elb_access_log
+group by
+  request_date
+order by
+  request_date asc;
+```
+
+### Top 10 clients by request volume
+
+List the top 10 client IP addresses making requests.
+
+```sql
+select
+  client_ip,
+  count(*) as request_count
+from
+  aws_elb_access_log
+group by
+  client_ip
+order by
+  request_count desc
+limit 10;
+```
+
+### Request distribution by target
```

Comment:
The query title/description don't align with what the query is looking for, i.e., why are we checking for avg processing time in the query if the desc is talking about request distribution?